frontend/management/src/containers/applicant-personal/ApplicantPersonalPageContainer.tsx

```diff
@@ -29,6 +29,13 @@ function ApplicantPersonalPageContainer(): ReactNode {
     }
   }, [id]);
 
+  // useEffect((): void => {
```

Comment:
Do we still need this? Usually we leave a comment if commented section will be used later.